packages/dappmanager/src/initializeDb.ts

```diff
@@ -71,11 +72,17 @@ export default async function initializeDb(): Promise<void> {
   //   This boolean will trigger the VPN nodejs process to try to open the ports with UPnP
   //   UPnP is available and necessary only if the internalIp is not equal to the public IP
   //   and the external IP from UPnP command succeeded
-  const upnpAvailable = publicIp
-    ? Boolean(externalIp && internalIp !== publicIp) && (await upnpc.list())
+  const upnpAvailable =
+    Boolean(publicIp && externalIp && internalIp !== publicIp) &&
+    upnpc
+      .list()
+      .then(() => true)
+      .catch(e => {
+        if (e instanceof UpnpError) return e.isUpnpAvailable;
+        return false;
+      })
       ? true
```

Comment:
This will always be true because you are not awaiting the promise. Create a helper function that returns `Promise<boolean>`, will make the code more readable